includes/Title.php

```diff
@@ -4281,7 +4281,8 @@ public function countAuthorsBetween( $old, $new, $limit ) {
 			return 0; // nothing to compare
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)